js/templates/buyWizard.html

```diff
@@ -28,10 +28,12 @@ <h3 class="custCol-text padding30"><%= polyglot.t('CheckExchanges') %> </h3>
         </div>
         <div class="txt-bright pad20">
           <div class="txt-center row10 textSize20px custCol-text">
-            <%= polyglot.t('buyFlow.SendBTCtoAddress', {amount: '<span class="js-buyWizardDetailsTotalBTC"></span>'}) %>
-          </div>
-          <div class="txt-center row10">
-            <a class="textSize16px js-buyWizardPayLink custCol-text"><span class="js-buyWizardPayURL"></span></a>
+            <%=
+              polyglot.t('buyFlow.SendBTCtoAddress', {
+                amount: '<span class="js-buyWizardDetailsTotalBTC"></span>',
+                recipient: '<a class="textSize16px js-buyWizardPayLink custCol-text"><span class="js-buyWizardPayURL"></span></a>'
```

Comment:
This could get a little tricky, since our translators may not know html, and changing the order of where the recipient element is placed could mess up the layout.